apps/docs/.vitepress/config.ts

```diff
@@ -176,6 +176,21 @@ export default defineConfig({
               },
             ],
           },
+          {
+            text: 'Wallet Manager',
+            link: '/guide/wallet-manager/',
+            collapsed: true,
+            items: [
+              {
+                text: 'Getting Started With WalletManager',
```

Comment:
```suggestion
                text: 'Getting Started With Wallet Manager',
```

This should fix the build